packages/react/src/components/Tabs/Tab.tsx

```diff
@@ -2,19 +2,25 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 interface TabProps extends React.HTMLAttributes<HTMLLIElement> {
+  targetref: React.RefObject<HTMLDivElement>;
```

Comment:
This might be considered nitpicky, but I do think it's important we try to maintain a consistent API across components.

```suggestion
  target: React.RefObject<HTMLDivElement> | HTMLElement;
```

This keeps it consistent with other component apis (such as [Tooltip](https://github.com/dequelabs/cauldron/blob/8f02a508ac963edc8ff33e1ae580d2f6d55c53e4/packages/react/src/components/Tooltip/index.tsx#L13)) that need references to other elements to maintain associations.